### PR TITLE
Add new code owner

### PR DESCRIPTION
**Pull request by vitormattos**

### CODE REVIEW

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
I'm just the backup :)

**Review by ChristophWurst — APPROVED** [review]
Squash and merge

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*	@ChristophWurst
+*	@ChristophWurst @vitormattos
```
[the remark this answers is not in the document]
Comment (vitormattos) [reply .github/CODEOWNERS]: Done, but only to know more: What this order affect? Looking at this documentation I didn't found info about the order:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply .github/CODEOWNERS]: I don't think it matters for Github

**Pull request closed without merging**